packages/sdk-js/README.md

```diff
@@ -4,86 +4,20 @@
 [![Lint and Test](https://github.com/KILTprotocol/sdk-js/workflows/Lint%20and%20Test/badge.svg)](https://github.com/KILTprotocol/sdk-js/actions/workflows/tests.yml)
 [![Compatible with latest dependencies](https://github.com/KILTprotocol/sdk-js/actions/workflows/tests-polkadot-deps.yml/badge.svg?event=schedule)](https://github.com/KILTprotocol/sdk-js/actions/workflows/tests-polkadot-deps.yml)
 
-Data sovereignty and interoperability
-
 # KILT JavaScript / TypeScript SDK
 
-The open-source KILT SDK is written in TypeScript and enables you to quickly and easily build dApps around new business use cases. KILT is a protocol for self-sovereign data and interoperability built on top of the permissionless KILT blockchain. The SDK provides collection of classes and methods you can utilize to interact with the KILT Protocol.
+The open-source KILT SDK is written in TypeScript and enables developers to quickly and easily build dApps around new business use cases.
+KILT is a protocol for self-sovereign data and interoperability built on top of the permissionless KILT blockchain.
+The SDK provides collection of classes and methods to interact with the KILT Protocol.
 
 - **Self-sovereign data.** Have ownership of your digital and analog identities, with control over who your users share that data with. Providing that extra layer of flexiblity and security.
 - **Interoperability.** Claim Types (CTYPEs) facilitate the adoption of standardised credential content structures.
 
-To learn more, checkout the KILT [Whitepaper](https://kilt.io/wp-content/uploads/2019/05/KILT-Whitepaper-v2019-May-28.pdf) or see our [how it works](https://kilt.io/kilt-data-sovereignty-and-interoperability/) video.
-Regarding the privacy enhancement, please have a look at our [lightning talk for Sub0 April 2020](https://drive.google.com/file/d/16HHPn1BA5o-W8QCeHfoTI1tNb5yQUZzt/view?usp=sharing).
-
 ## Documentation
 
-> To avoid confusion between the latest released SDK version and the `develop` default branch, the links in the list **point to the master branch**, which always contains the latest official release of the SDK.
-
-[KILT](https://kilt.io) documentation is provided in several guides and demos.
-
-- [KILT workshop](https://dev.kilt.io/docs/sdk/workshop/welcome/) 👈 Start here to get familiar with the basics
-- [Getting started guide](https://github.com/KILTprotocol/sdk-js/blob/master/docs/getting-started.md) 👈 Start here if you'd like to include KILT in your project
-- [KILT Developer docs](https://dev.kilt.io/) 👈 Checkout for in-depth documentation of KILT features.
-- [API documentation](https://kiltprotocol.github.io/sdk-js)
-- [Demo client](https://kilt.io/developers-sub/kilt-demo-client/)
-- [Demo client code](https://github.com/KILTprotocol/demo-client)
-
-To help improve, please see our [contribution page](/docs/contribution-guide.md).
-
-## How to install the SDK
```

Comment:
I would keep the how to install and embed here, since for js libraries that is best practice to have that here.

Reply:
Alright. https://github.com/KILTprotocol/sdk-js/pull/540/commits/512a5a3e965c8d3f15574aee7c03557a272d6680 and https://github.com/KILTprotocol/sdk-js/pull/540/commits/d68d385149423d8a5fe1889ccc22811a60e5f2f4 should do it.